tool/forestadmin/docker-compose.yml

```diff
@@ -1,15 +1,17 @@
 version: '3'
 services:
-  app:
+  forestadmin:
     build:
       context: .
       dockerfile: Dockerfile
+    env_file:
+      - .env
```

Comment:
I think this is implicit, can you check, because, if it's automatic, I prefer that you remove it in order to allow people to use a .env file or env variables